Keymaps/Default (OSX).sublime-keymap

```diff
@@ -8,64 +8,319 @@
     //         "text": "LSP: ",
     //     }
     // },
-
+    // Override native save to handle Code-Actions-On-Save
+    {
+        "command": "lsp_save",
+        "keys": [
+            "super+s"
+        ],
+        "context": [
+            {
+                "key": "lsp_session_with_capability",
+                "operator": "equal",
+                "operand": "codeActionProvider"
```

Comment:
> Is it possible to be more specific here? For CAOS it would be more specific to check codeActionProvider.codeActionKinds.

No only top-level keys are supported.

> do you have an idea how we'll handle check here once we hook up formatting onto lsp_save also?

```js
"operand": "codeActionProvider | documentFormattingProvider | documentRangeFormattingProvider"
```